reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt

```diff
@@ -45,6 +45,13 @@ import org.ossreviewtoolkit.utils.toHexString
 private const val DEFAULT_TEMPLATE_ID = "basic-pdf-template"
 private const val TEMPLATE_ID = "template.id"
 private const val TEMPLATE_PATH = "template.path"
+private const val TEMPLATE_COVER_PDF = "cover.pdf"
```

Comment:
Nit: Let's have a blank line before this one to separate the cover-related constants from the others.